addOns/exportreport/src/main/java/org/zaproxy/zap/extension/exportreport/ExtensionExportReport.java

```diff
@@ -80,7 +80,10 @@
     private static final int ARG_SOURCE_INFO_IDX = 1;
     private static final int ARG_ALERT_SEVERITY_IDX = 2;
     private static final int ARG_ALERT_DETAILS_IDX = 3;
-    private CommandLineArgument[] arguments = new CommandLineArgument[4];
+    private static final int ARG_SCAN_ID_IDX = 4;
```

Comment:
I'm not sure we should provide this command line argument (yet?). This would not work properly as ZAP does not persist the scan state/IDs.

Reply:
I see, it currently does nothing as all scans are retrieved in case of an error retrieving the scan.  
The other argument (for passive alert inclusion) also doesn't work in this implementation, does ZAP not persist the alert's source? Should that code simply be commented out and a new issue be created to link to it?

Reply:
The source is persisted, that argument Is working fine here. Could you provide more details?

I think it's fine commenting the `scan_id` for now.

Reply:
You're right, I mistook some active scan results for passive alerts, sorry. I can confirm it seems to be working fine.